Graph-Convolutional-Networks-Model-Relations-In-Data/README.md

```diff
@@ -0,0 +1,4 @@
+# Training
+For training run [jupyter notebook](graph_convolutional_networks_model_relations_in_data.ipynb)
+
+_Note: we use dataset from_ [_this post_](../PyTorch-Multi-Label-Image-Classification-Image-Tagging/README.md)
```

Comment:
@VictorBebnev, do we need to follow any steps from that post in order to make it work? I'd suggest to write it explicitly. 
> Note: we use the dataset from this post, so please refer to it for a detailed description and instructions.